apps/ui/src/Communities/CommunitiesFeed.tsx

```diff
@@ -168,7 +170,7 @@ const CommunitiesFeed: React.FC<CommunitiesFeedProps> = ({ communities }) => {
           <CommunityCard
             key={index}
             community={community}
-            userStatus={userCommunitiesStatus[community.id]}
+            userStatus={userCommunitiesStatus[community.id] as UserStatus}
```

Comment:
consider make the userStatus optional in the CommunityCard props & then pass userCommunitiesStatus[community.id] (if its undifiend then CommunityCard knows the user can request to join the community)